Tests/Controller/HelperControllerTest.php

```diff
@@ -321,8 +321,6 @@ public function testappendFormFieldElementAction($validatorInterface)
         $formExtension = $this->getMock('\Twig_ExtensionInterface', array('renderListElement', 'initRuntime', 'getTokenParsers', 'getNodeVisitors', 'getFilters', 'getTests', 'getFunctions', 'getOperators', 'getGlobals', 'getName'));
 
         $formExtension->expects($this->once())->method('getName')->will($this->returnValue('form'));
-        $formExtension->expects($this->never())->method('searchAndRenderBlock');
-        $formExtension->expects($this->never())->method('setTheme');
```

Comment:
Yeah i did confuse it with another interface (`Symfony\Bridge\Twig\Form\TwigRendererInterface`) at first to.